### ISSUE TITLE
Code backpropagation algorithm for MLP

### ISSUE THREAD

**Issue by pparuzel** [reconndev/neural-network#5]
* Do math for Gradient Descent
* Correct NN Library if needed
* Code it
